lib/model/instance/assignment.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/opendatakit/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+
+const Instance = require('./instance');
+const { ExtendedInstance, HasExtended } = require('../trait/extended');
+const { ActeeSpeciesTrait } = require('../trait/actee');
+
+const ExtendedAssignment = ExtendedInstance({
+  fields: { readable: [ 'actor', 'roleId' ] }
```

Comment:
Does `ExtendedAssignment` need a `forApi()` method? I'm noticing that each `actor` property in the `/projects/:id/assignments` response has `acteeId`, `meta`, and `deletedAt` properties, which I think we usually don't return for actors.